Plan/common/src/main/resources/assets/plan/web/js/color-selector.js

```diff
@@ -71,7 +67,11 @@
     // Change the color of the theme
     setColor(selectedColor ? selectedColor : themeDefaultColor);
 
-    let nightMode = window.localStorage.getItem('nightMode') == 'true';
+    if (selectedColor === null) {
```

Comment:
Is this possible to move this inside `setColor `function instead?

The other `setItem` call is inside that function so this appears like a leak of functionality from that function